<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282828;
color: #ffffff;</string>
  </property>
  <widget class="QPushButton" name="pauseResumeBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Start page:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startPage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="randomBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop condition:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="languageComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search order:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bfsRadio">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="dfsRadio">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay (s):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delay">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startStopBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reachPageCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reach page</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="maxPageCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Max page number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="maxDepthCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Max depth</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="timeLimitCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Time limit (s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reachPage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxPage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxDepth">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLimit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="randomRadio">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Load detailed info:</string>
   </property>
  </widget>
  <widget class="QLabel" name="switchContainer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
